static/src/components/sprint/sprint.ts

```diff
@@ -124,7 +126,7 @@ class Sprint extends Control {
         return randomSort(await Words.checkWords(words, group, page)) as IWord[];
       }
 
-      const randomPage = randomSort([...Array(bookConfig.maxPage).keys()]).slice(0, 5) as number[];
+      const randomPage = randomSort([...Array(bookConfig.maxPage).keys()]).slice(0, 15) as number[];
```

Comment:
Magic number. Please move this value in meaningful constant

Reply:
I think I should remove this page limit and load everything in random order